src/CodeStyle/CSharp/Analyzers/xlf/CSharpCodeStyleResources.cs.xlf

```diff
@@ -2,21 +2,91 @@
         <source>Indentation preferences</source>
         <target state="translated">Předvolby odsazení</target>
         <note />
       </trans-unit>
+      <trans-unit id="Make_readonly_fields_writable">
+        <source>Make readonly fields writable</source>
+        <target state="new">Make readonly fields writable</target>
+        <note>{Locked="readonly"} "readonly" is C# keyword and should not be localized.</note>
+      </trans-unit>
+      <trans-unit id="Null_checking_preferences">
+        <source>Null-checking preferences</source>
+        <target state="new">Null-checking preferences</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Pattern_matching_preferences">
+        <source>Pattern matching preferences</source>
+        <target state="new">Pattern matching preferences</target>
+        <note />
+      </trans-unit>
       <trans-unit id="Space_preferences">
         <source>Space preferences</source>
         <target state="translated">Předvolby mezer</target>
         <note />
       </trans-unit>
+      <trans-unit id="Use_explicit_type">
+        <source>Use explicit type</source>
+        <target state="new">Use explicit type</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Use_explicit_type_instead_of_var">
+        <source>Use explicit type instead of 'var'</source>
+        <target state="new">Use explicit type instead of 'var'</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Use_implicit_type">
+        <source>Use implicit type</source>
+        <target state="new">Use implicit type</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Use_switch_expression">
+        <source>Use 'switch' expression</source>
+        <target state="new">Use 'switch' expression</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="Using_directive_is_unnecessary">
+        <source>Using directive is unnecessary.</source>
+        <target state="new">Using directive is unnecessary.</target>
+        <note />
+      </trans-unit>
       <trans-unit id="Wrapping_preferences">
         <source>Wrapping preferences</source>
         <target state="translated">Předvolby zalamování</target>
         <note />
       </trans-unit>
+      <trans-unit id="use_var_instead_of_explicit_type">
+        <source>use 'var' instead of explicit type</source>
+        <target state="new">use 'var' instead of explicit type</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="using_directive_preferences">
+        <source>'using' directive preferences</source>
+        <target state="new">'using' directive preferences</target>
+        <note>{Locked="using"} "using" is a C# keyword and should not be localized.</note>
+      </trans-unit>
+      <trans-unit id="var_preferences">
+        <source>var preferences</source>
+        <target state="new">var preferences</target>
+        <note />
+      </trans-unit>
```

Comment:
will tihs be a pain point for loc?  or will their tools auto migrate things?

Reply:
I think their tool has a backing db that will auto-migrate the translations. I think we still want to potentially create separate shared resource files for resources shared across projects. I tried that but wasn't able to get the .desiger.cs auto-generation hooked up to auto-update it when the underlying resx changes.